tests/runtime_neutral/test_router_bridge.py

```
@@ -40,6 +41,40 @@ def run_bridge(prompt: str, grade: str, task_type: str, requested_skill: str | N
+        str(REPO_ROOT / "scripts" / "router" / "resolve-pack-route.ps1"),
+        "-Prompt",
+        prompt,
+        "-Grade",
+        grade,
+        "-TaskType",
+        task_type,
+    ]
+    if requested_skill:
+        command.extend(["-RequestedSkill", requested_skill])
+
+    completed = subprocess.run(
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)